tests/test_env.py

```diff
@@ -152,3 +162,265 @@ def test_gen_req_file(tmpdir):
         'lorem == 0.1.1, > 0.1 ; python_version < "3.7" and platform_machine == "x86_64" and platform_system == "Linux"'
         in req_lines
     )
+
```

Comment:
Please add doc string to test cases that don't have one.

Reply:
Done